taxi/views.py

```diff
@@ -36,6 +46,21 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        contexte = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name", "")
+        contexte["search_form"] = ManufacturerSearchForm(
+            initial={"name": name}
+        )
+        return contexte
```

Comment:
There is a typo in the variable name `contexte`. It should be `context` to match the expected naming convention.